cms/envs/common.py

```diff
@@ -917,6 +917,10 @@
 SESSION_SERIALIZER = 'openedx.core.lib.session_serializers.PickleSerializer'
 SESSION_COOKIE_DOMAIN = ""
 SESSION_COOKIE_NAME = 'sessionid'
+S3URL_PLUGIN_LINKTYPES = "Download,Offer"
```

Comment:
It might be neater to have a NODE_ENV_VARIABLES (or something else with a better name) variable instead which can hold all these in JSON form, and pass all of them to js. Then it could become more generic and this part could be merged upstream.

Reply:
Good idea @mavidser. I changed it to `ADDITIONAL_NODE_ENV_VARS`.